nbbuild/antsrc/org/netbeans/nbbuild/ModuleListParser.java

```diff
@@ -126,7 +126,7 @@ public static void resetCaches() {
                             while (tok2.hasMoreTokens()) {
                                 String module = tok2.nextToken();
                                 standardModules.add(module);
-                                doFastScan |= new File(root, module.replace('/', File.separatorChar)).equals(basedirF);
+                                // doFastScan |= new File(root, module.replace('/', File.separatorChar)).equals(basedirF);
```

Comment:
Prevents a `ClassCastException` when deserializing `nb-*-cache*.ser` which I found in the logs.